Tests/Fixtures/TestExtensions/test_extension/Classes/Controller/MainController.php

```diff
@@ -82,7 +82,7 @@ public function createAction(\FIXTURE\TestExtension\Domain\Model\Main $newMain)
      * action edit
      *
      * @param \FIXTURE\TestExtension\Domain\Model\Main $main
-     * @ignorevalidation $main
+     * @TYPO3\CMS\Extbase\Annotation\IgnoreValidation ("main")
```

Comment:
Did you test whether this works? Are spaces really allowed here?

Reply:
https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.0/Feature-83094-ReplaceIgnorevalidationWithTYPO3CMSExtbaseAnnotationIgnoreValidation.html?highlight=ignorevalidation
does not have an example with a space at least

Reply:
No, I didn't check the MainController.php in Tests.
Yes, I know, the reference does not have an example with a space. 
I added the space to pass the check. I don't have any idea, why it didn't pass the check without the space.

Reply:
The solution is not to add a space to the test controller annotation, but to change the arguments in ->setTag().
If you provide 2 arguments, the first is the tag name and the second is the tag value which result in a annotation with a separated space, like in @var $foo etc.
So what we need here is just provide a tag name and no value:
 $actionMethod->setTag(
     'TYPO3\CMS\Extbase\Annotation\IgnoreValidation' . '("' . $parameterName . '")' 
); (point instead comma)

Reply:
@nicodh thanks for your explanation! Changed it accordingly.